docs/sdk/dataproxy-sdk/http.md

```diff
@@ -3,9 +3,28 @@ title: HTTP Report
 sidebar_position: 3
 ---
 
+## HTTP report process introduction
+InLong processes HTTP report through the DataProxy node. The data source regularly obtains the access point list from the Manager, then selects available HTTP DataProxy nodes from the access point list according to its own policies, and then uses the HTTP protocol for data production; due to the performance of HTTP report due to problems such as low proportion of valid data and easy loss of request messages, it is recommended that users use TCP to report data as much as possible. The overall HTTP report process is shown below:

```

Comment:
> due to the performance of HTTP report due to problems such as low proportion of valid data and easy loss of request messages

Suggestion : 
due to the issues of low performance, low percentage of effective data, and easy loss of request messages in HTTP reporting.